roles/splunk_common/files/Splunkd.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+After=network.target
+
+[Service]
+Type=simple
+Restart=always
+ExecStart=/opt/splunk/bin/splunk _internal_launch_under_systemd
```

Comment:
Let's turn this into a Jinja template so we can accodomate variables + support for UF

Reply:
In a separate PR perhaps?